src/get-started/codelab.md

```diff
@@ -380,6 +380,8 @@ a child inside the existing `MyApp` stateless widget.
 
   ```dart
 class RandomWords extends StatefulWidget {
+  const RandomWords();
```

Comment:
I'd prefer the full `const RandomWords({Key? key): super(key:key);` incantation here.

Reply:
Done!